src/pages/Checkout/CheckoutPage.vue

```diff
@@ -422,7 +423,8 @@ export default {
 						client.query({ query: initializeCheckout, fetchPolicy: 'network-only' }),
 						client.query({ query: upsellQuery }),
 						client.query({ query: experimentQuery, variables: { id: 'require_deposits_matched_loans' } }),
-						client.query({ query: experimentQuery, variables: { id: 'upsells_copy' } })
+						client.query({ query: experimentQuery, variables: { id: 'upsells_copy' } }),
+						client.query({ query: experimentQuery, variables: { id: 'upsells_dynamic' } }),
```

Comment:
This needs to match the other setting settings ;) I think it should be `dynamic_upsells`